src/styles/theme.js

```diff
@@ -0,0 +1,95 @@
+import { css } from "styled-components";
+
+const breakpoints = {
+  tablet: 1040,
+  mobile: 640
+};
+
+const media = Object.keys(breakpoints).reduce((acc, label) => {
+  acc[label] = (...args) => css`
+    @media screen and (max-width: ${breakpoints[label] / 16}em) {
+      ${css(...args)}
+    }
+  `;
```

Comment:
is it mixin?

Reply:
More like a way of autogenerating a mixin. You can see how to use it in `Button/index.js`.
It's based upon Styled Components docs.

https://www.styled-components.com/docs/advanced#media-templates

Reply:
okey, cool 👍